src/constants/general.ts

```diff
@@ -68,4 +68,5 @@ export default {
     DISABLED: 'disabled',
     DEFAULT_ON: 'default_on',
     DEFAULT_OFF: 'default_off',
+    DEFAULT_GROUP: 'board',
```

Comment:
What's `'board'`?

Reply:
> What's `'board'`?

@mkraft - It is the name of a dummy group (note: DEFAULT_GROUP is used in websocket.test.js only,). There doesn't seem to be a separate file for redux constants for test only, so we can either create one or give this a different name like DEFAULT_TEST_GROUP?

Reply:
Ok, it is the test 'board'. It threw me off that it was in a non-test file. 0/5 from my perspective.